presto-main/src/main/java/com/facebook/presto/metadata/Metadata.java

```diff
@@ -454,6 +455,15 @@
      */
     List<GrantInfo> listTablePrivileges(Session session, QualifiedTablePrefix prefix);
 
+    /**
+     * Gets the materialized views for a give table
+     */
+    @Experimental
```

Comment:
Why experimental?

Also should we keep it consistent with other materialized view methods and have no implementation here?